examples/projects/groupproject.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: projects.gitlab.crossplane.io/v1alpha1
+kind: Project
+metadata:
+  name: example-group-project

```

Comment:
same as with the subgroup example, since this is just a `project` my preference is to omit this example. People can take a look at the API documentation at gitlab or doc.crds.dev/crossplane-contrib/provider-gitlab (once it is released) to find all features.

Reply:
Removed examples/projects/groupproject.yaml